README.rst

```diff
@@ -0,0 +1,72 @@
+.. BADGES
+   ======
+.. image:: https://travis-ci.org/tgsmith61591/skutil.svg?branch=master
+   :target: https://travis-ci.org/tgsmith61591/skutil
+.. image:: https://coveralls.io/repos/github/tgsmith61591/skutil/badge.svg?branch=master
+   :target: https://coveralls.io/github/tgsmith61591/skutil?branch=master
+.. image:: https://img.shields.io/badge/python-2.7-blue.svg
+   :target: https://img.shields.io/badge/python-2.7-blue.svg
+.. image:: https://img.shields.io/github/release/tgsmith61591/skutil.svg
+   :target: https://img.shields.io/github/release/tgsmith61591/skutil
+.. image:: https://img.shields.io/github/issues-raw/tgsmith61591/skutil.svg
+   :target: https://github.com/tgsmith61591/skutil/issues
+.. image:: https://img.shields.io/github/issues-pr-raw/tgsmith61591/skutil.svg
```

Comment:
Again, this is an easily accessible #. I don't know that we should clutter the badge space with this.